data/xml/2008.amta.xml

```diff
@@ -0,0 +1,543 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2008.amta">
+  <volume id="keynotes" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Keynote Presentations</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Waikiki, USA</address>
+      <month>October 21-25</month>
+      <year>2008</year>
+    </meta>
+    <paper id="1">
+      <title>The Triple-Advantage Factor of <fixed-case>MT</fixed-case>: Cost, Time-to-Market, and <fixed-case>FAUT</fixed-case></title>
+      <author><first>Will</first><last>Burgett</last></author>
+      <author><first>Julie</first><last>Chang</last></author>
+      <url hash="eb9b0b20">2008.amta-keynotes.1</url>
+      <pages>1-10</pages>
+    </paper>
+    <paper id="2">
+      <title>Bringing humans into the loop: Localization with <fixed-case>MT</fixed-case> at Traslán</title>
+      <author><first>Declan</first><last>Groves</last></author>
+      <url hash="9881c9aa">2008.amta-keynotes.2</url>
+      <pages>11-22</pages>
+    </paper>
+    <paper id="3">
+      <title>Machine Translation 2008: Science meets Solution</title>
+      <author><first>Mark</first><last>Tapling</last></author>
+      <url hash="b554a117">2008.amta-keynotes.3</url>
+      <pages>23-36</pages>
+    </paper>
+  </volume>
+  <volume id="papers" ingest-date="2021-05-19">
+    <meta>
+      <booktitle>Proceedings of the 6th Conference of the Association for Machine Translation in the Americas: Technical Papers</booktitle>
```

Comment:
Eighth Conference

Reply:
Also, where did you get "Technical Papers" from? From the website, I might put "Research Papers"

Reply:
I chose "Technical Papers" to be consistent with previous volumes. I'll switch to "Research Papers".